packages/apollo-link-error/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ### vNEXT
 
+### 1.0.10
+- Pass `forward` into error handler for ErrorLink to support retrying a failed request 
```

Comment:
Normally we place the changelog entry under `vNext` and then when we publish, we move them to the appropriate place. This is because the next version could be a minor version, rather than a patch, depending on the features that we add.

For a deeper look into how we normally version things, check out the [semver spec](https://semver.org/)

Reply:
Fixed